frontend/src/app/components/block/block.component.html

```diff
@@ -114,7 +114,7 @@ <h1>
                   <td><span class="skeleton-loader"></span></td>
                 </tr>
               </ng-template>
-              <tr>
+              <tr *ngIf="network !== 'liquid' && network !== 'liquidtestnet'">
```

Comment:
This fix is correct, other changes must be reverted. See https://github.com/mempool/mempool/pull/1707